_posts/2020-05-07-may-meetup.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "May 2020 - Remote again! 💪🏼"
+date:   2020-04-29 19:00:00
```

Comment:
this should be in the past to make sure it's actually included in the generated html (the preview doesn't currently include it)